pkgs/development/compilers/llvm/common/libc/default.nix

```diff
@@ -79,12 +79,8 @@ stdenv.mkDerivation (finalAttrs: {
       (lib.cmakeFeature "LLVM_ENABLE_RUNTIMES" "libc;compiler-rt")
       # Tests requires the host to have a libc.
       (lib.cmakeBool "LLVM_INCLUDE_TESTS" (stdenv.cc.libc != null))
-      (lib.cmakeBool "LLVM_LIBC_INCLUDE_SCUDO" true)
-      (lib.cmakeBool "COMPILER_RT_BUILD_SCUDO_STANDALONE_WITH_LLVM_LIBC" true)
-      (lib.cmakeBool "COMPILER_RT_BUILD_GWP_ASAN" false)
```

Comment:
This is something specific to scudo?

Reply:
But there are four options being removed here? One is `COMPILER_RT_BUILD_GWP_ASAN, which is the line I commented on, since it doesn't have "SCUDO" in its name like the others.

Reply:
I see 3 options being removed. It's a part of compiler-rt and affects scudo. When we disable scudo, a warning throws up that the option is undefined.

Reply:
> I see 3 options being removed.

There are four.  Remember that the GitHub UI shows you the line I placed my comment on and a few above it, none below.  (I think I remember this catching you out before and leading to another very confusing conversation…)

1. LLVM_LIBC_INCLUDE_SCUDO
2. COMPILER_RT_BUILD_SCUDO_STANDALONE_WITH_LLVM_LIBC
3. COMPILER_RT_BUILD_GWP_ASAN
4. COMPILER_RT_SCUDO_STANDALONE_BUILD_SHARED

> When we disable scudo, a warning throws up that the option is undefined.

That answers my question, thanks.

Reply:
Oh lol :melting_face: